src/index.css

```
@@ -284,19 +370,55 @@
+            var(--accent-gradient-start, var(--accent-500)),
+            var(--accent-gradient-end, var(--accent-400))
+        );
+        -webkit-background-clip: text;
+        background-clip: text;
+        -webkit-text-fill-color: transparent;
+        color: transparent;
+    }
+
     .text-gradient-emerald {
         background: none;
         -webkit-text-fill-color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)